dev/com.ibm.ws.collector/src/com/ibm/ws/collector/internal/TaskImpl.java

```diff
@@ -68,6 +68,11 @@ public void run() {
                 } catch (InterruptedException exit) {
                     //Exit
                     done = true;
+                } catch (NullPointerException e) {
```

Comment:
You need to add `NullPointerException` to the `@FFDCIgnore()` annotation of this method.